Jenkinsfile

```diff
@@ -0,0 +1,73 @@
+def label = "cognite-grafana-datasource-${UUID.randomUUID().toString().substring(0, 5)}"
+def imageName = "cognite/grafana-cdp"
+def devImageName = "cognite/grafana-cdp-dev"
+
+podTemplate(
+  label: label,
+  containers: [
+    containerTemplate(
+      name: 'node',
+      image: 'node:9',
+      ttyEnabled: true
+    ),
+    containerTemplate(
+      name: 'docker',
+      command: '/bin/cat -',
+      image: 'docker:18.06.1-ce',
+      resourceRequestCpu: '100m',
+      resourceRequestMemory: '500Mi',
+      resourceLimitCpu: '300m',
+      resourceLimitMemory: '500Mi',
+      ttyEnabled: true
+    ),
+  ],
+  volumes: [
+    secretVolume(secretName: 'npm-credentials',
+                 mountPath: '/npm-credentials',
+                 readOnly: true),
+    secretVolume(secretName: 'cognitecicd-dockerhub', mountPath: '/dockerhub-credentials'),
+    hostPathVolume(hostPath: '/var/run/docker.sock', mountPath: '/var/run/docker.sock'),
+  ],
+  envVars: [
+    envVar(key: 'CHANGE_ID', value: env.CHANGE_ID),
+  ],
+  ) {
+    node(label) {
+      def shortSha
+      container('node') {
+        stage('Checkout') {
+          checkout(scm)
+          shortSha = sh(returnStdout: true, script: 'git rev-parse --short HEAD').trim()
+        }
+
+        stage('Prepare') {
```

Comment:
Build and test, or keep `'Prepare'` but then it should just do `yarn` and not `yarn build` (which would then be in a `'Build'` stage)

Reply:
Good point.